sympy/algebras/quaternion.py

```diff
@@ -720,3 +719,409 @@ def to_rotation_matrix(self, v=None):
+
+        Explanation
+        ===========
+
+        If q is a quaternion given by q = a + b*i + c*j + d*k where a, b, c and d
+        are real numbers then the angle of the quaternion is given by
+
+            angle := atan2 (\sqrt(b^2+c^2+d^2), a)
+
+        Returns
+        =======
+
+        expr: The angle of the quaternion.
+
+        Examples
+        ========
+
+        >>> from sympy.algebras.quaternion import Quaternion
+        >>> q = Quaternion(1, 4, 4, 4)
+        >>> q.angle()
+        atan(4*sqrt(3))
+
+        >>> q = Quaternion(0, 8, 13, 12)
+        >>> q.angle()
+        pi/2
+
+        """
+
+        return atan2(self.vector_part().norm(), self.scalar_part())
+
+    @classmethod
+    def coplanar(cls, q1, q2):
+        """
+        Returns True if the plane of the two quaternions are coplanar else returns False.
+
+        Explanation
+        ===========
+
+        checks if the 3D vectors written as quaternions are in the same plane.
+        In order to return a True, the quaternions and the origin should be in the
+        same plane, and the real part must be zero for it to make sense.
+
+        Parameters
+        ==========
+
+        q1 : a Quaternion
+        q2 : a Quaternion
+
+        Examples
+        ========
+
+        >>> from sympy.algebras.quaternion import Quaternion
+        >>> q1 = Quaternion(1, 4, 4, 4)
+        >>> q2 = Quaternion(2, 8, 8, 8)
+        >>> Quaternion.coplanar(q1, q2)
+        True
+
+        >>> q1 = Quaternion(2, 8, 13, 12)
+        >>> Quaternion.coplanar(q1, q2)
+        False
+
+        See Also
+        ========
+
+        is_pure
+
+        """
+        if (q1.is_zero_quaternion()) or (q2.is_zero_quaternion()):
+            raise ValueError('Any of the provided quaternions must not be 0')
+
+        aq = q1.axis()
+        ar = q2.axis()
+
+        return (aq == ar or aq == -ar)
+
+    @classmethod
+    def ternary_coplanar(cls, q1, q2, q3):
+        """
+        Returns True if the axis of the pure quaternions seen as 3D vectors
+        q1, q1, and q2 are coplanar else returns False.
+
+        Explanation
+        ===========
+
+        Two general quaternions are coplanar (in this arc sense) when their axes are parallel.
+
+        Parameters
+        ==========
+
+        q1 : a pure Quaternion.
+        q2 : a pure Quaternion.
+        q3 : a pure Quaternion.
+
+        Examples
+        ========
+
+        >>> from sympy.algebras.quaternion import Quaternion
+        >>> q1 = Quaternion(0, 4, 4, 4)
+        >>> q2 = Quaternion(0, 8, 8, 8)
+        >>> q3 = Quaternion(0, 24, 24, 24)
+        >>> Quaternion.ternary_coplanar(q1, q2, q3)
+        True
+
+        >>> q1 = Quaternion(0, 8, 16, 8)
+        >>> q2 = Quaternion(0, 8, 3, 12)
+        >>> Quaternion.ternary_coplanar(q1, q2, q3)
+        False
+
+        """
+
+        if not q2.is_pure() or not q3.is_pure() or not q1.is_pure():
+            raise ValueError('The provided quaternions must be pure')
```

Comment:
You need to be careful using `not` with three-valued logic because `not None` gives `True`. That's why we have `fuzzy_not`.